### ISSUE TITLE
Important question

### ISSUE THREAD

**Issue by DoubleDim** [nickvane/Magento-RestApi#53]
Does this api works with Magento Community Edition v2? Coz im trying to connect with my demostore on localhost but i keep getting: "Unable to load admin page: 'http://localhost/magento/admin_erfitc'" exception. Here is my code:

private void btnRead_Click(object sender, EventArgs e)
        {
            var client = new MagentoApi()
            .SetCustomAdminUrlPart("admin_erfitc")
            .Initialize("http://localhost/magento", "ConsumerKey", "ConsumerSecret")
            .AuthenticateAdmin("username", "password"); //writing my admin panel login info

![magec](https://cloud.githubusercontent.com/assets/23503889/23169792/bd1b96f0-f855-11e6-8345-737d9a3e491f.jpg)

**Comment by NotAndD** [comment]
I tried and unfortunately, it doesn't work, it needs massive modifies to start working because rest api urls and other stuff changed like a lot.

**Comment by DoubleDim** [comment]
Yes i was worrying about that because i was wasting time trying to figure out what im doin wrong but it seems that it's no more working after v2, thanks for your answer.
